pkg/kubelet/rkt/rkt.go

```diff
@@ -937,15 +985,21 @@ func (r *Runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, *k
 		newUnitOption("Service", "ExecStopPost", markPodFinished),
 		// This enables graceful stop.
 		newUnitOption("Service", "KillMode", "mixed"),
+		// Track pod info for garbage collection
+		newUnitOption(unitKubernetesSection, unitPodUID, string(pod.UID)),
```

Comment:
I don't have a problem with storing information in multiple locations in this case since it's useful for correlating distinct resources.